Tasks/app-store-promote/task.json

```diff
@@ -119,6 +119,38 @@
             "required": false,
             "groupName": "advanced",
             "helpMarkDown": "The name of your team if you are in multiple teams."
+        },
+        {
+            "name": "installFastlane",
+            "type": "boolean",
+            "label": "Install Fastlane",
+            "required": false,
+            "defaultValue": true,
+            "groupName": "advanced",
+            "helpMarkDown": "Select to install a version of the Fastlane tools.  Uncheck if your build machine already has the version of Fastlane to use."
+        },
+        {
+            "name": "fastlaneToolsVersion",
+            "type": "pickList",
+            "label": "Fastlane Version",
+            "defaultValue": "LatestVersion",
+            "options": {
+                "LatestVersion": "Latest Version",
+                "SpecificVersion": "Specific Version"
+            },
+            "groupName": "advanced",
+            "helpMarkDown": "Choose to install either the lastest version of fastlane or a specific version.",
```

Comment:
fastlane should actually be lower case - their logo also shows lowercase https://fastlane.tools/